public/docs/_examples/intro/dart/lib/logger_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.logger_service;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Injectable()
+class Logger {
+  void log(dynamic msg) {
+    window.console.log(msg);
```

Comment:
Another possibility (although some people object to using `=>` with methods that return void):

log(msg) => print(msg);
OR
void log(Object msg) => print(msg);